components/config/src/config/markup.rs

```diff
@@ -99,7 +101,7 @@ impl Markdown {
             return Ok((None, None));
         }
 
-        let mut ss = SyntaxSetBuilder::new();
+        let mut ss = SYNTAX_SET.clone().into_builder();
```

Comment:
i don't think this change is correct. `load_extra_syntaxes_and_highlight_themes` ends up in `extra_syntax_set`, which is used in `resolve_syntax_and_theme` here: https://github.com/getzola/zola/blob/434da458a38dd3e011ad147d410b61dac6e7570c/components/config/src/highlighting.rs#L44
and so this causes *all* syntaxes to be treated as Extra, not just the ones from `extra_syntaxes_and_themes`.